streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamWindowAggregateTest.java

```diff
@@ -49,112 +48,106 @@
     private KStreamTestDriver driver = null;
     private File stateDir = null;
 
+    @Before
+    public void setUp() throws IOException {
+        stateDir = TestUtils.tempDirectory("kafka-test");
+    }
+
     @After
-    public void tearDown() {
+    public void tearDown() throws Exception {
         if (driver != null) {
             driver.close();
+            driver = null;
         }
-        driver = null;
-    }
-
-    @Before
-    public void setUp() throws IOException {
-        stateDir = TestUtils.tempDirectory("kafka-test");
+        Utils.delete(stateDir);
     }
 
     @Test
     public void testAggBasic() throws Exception {
-        final File baseDir = Files.createTempDirectory("test").toFile();
-
-        try {
-            final KStreamBuilder builder = new KStreamBuilder();
-            String topic1 = "topic1";
-
-            KStream<String, String> stream1 = builder.stream(strSerde, strSerde, topic1);
-            KTable<Windowed<String>, String> table2 =
-                stream1.groupByKey(strSerde,
-                                   strSerde)
-                    .aggregate(MockInitializer.STRING_INIT,
-                               MockAggregator.TOSTRING_ADDER,
-                               TimeWindows.of(10).advanceBy(5),
-                               strSerde, "topic1-Canonized");
-
-            MockProcessorSupplier<Windowed<String>, String> proc2 = new MockProcessorSupplier<>();
-            table2.toStream().process(proc2);
-
-            driver = new KStreamTestDriver(builder, baseDir);
-
-            setRecordContext(0, topic1);
```

Comment:
Why are we deleting all these lines? And then reinserting? Is there a formatting change?